packages/core/src/render3/instructions/styling.ts

```diff
@@ -834,8 +834,18 @@ function findStylingValue(
     const containsStatics = Array.isArray(rawKey);
     // Unwrap the key if we contain static values.
     const key = containsStatics ? (rawKey as string[])[1] : rawKey;
-    let currentValue = key === null ? keyValueArrayGet(lView[index + 1], prop) :
-                                      key === prop ? lView[index + 1] : undefined;
+    let valueAtLViewIndex = lView[index + 1];
```

Comment:
What do you think about creating a var for this, to make it clear what it means
```suggestion
   const isStylingMap = key === null;
   let valueAtLViewIndex = lView[index + 1];
```

Reply:
fixed